### PR TITLE
Fix remaining forked_from after exercise renames

### PR DESCRIPTION
**Pull request by angelikatyborska**

### CODE REVIEW

File: exercises/concept/elyses-enchantments/.meta/config.json [hunk 1]
```diff
@@ -7,5 +7,5 @@
     "test": ["enchantments.spec.js"],
     "exemplar": [".meta/exemplar.js"]
   },
-  "forked_from": ["go/basic-slices"]
+  "forked_from": ["go/card-tricks"]
```
Comment (angelikatyborska) [inline exercises/concept/elyses-enchantments/.meta/config.json:10]: This one is still pending (https://github.com/exercism/go/pull/1464)

Comment (SleeplessByte) [reply exercises/concept/elyses-enchantments/.meta/config.json:10]: Should probably wait then until that's merged.

**Review by SleeplessByte — APPROVED** [review]

File: exercises/concept/elyses-enchantments/.meta/config.json (hunk 1 above)
Comment (angelikatyborska) [reply exercises/concept/elyses-enchantments/.meta/config.json:10]: Was merged :)

**Comment by SleeplessByte** [comment]
Thanks @angelikatyborska !!!

**Pull request merged**